src/openforms/utils/admin.py

```diff
@@ -49,3 +58,22 @@ def json_as_html(self, instance):
 
 admin.site.unregister(CSPReport)
 admin.site.register(CSPReport, CSPReportOverrideAdmin)
+
+
+class OutgoingRequestsLogOverrideAdmin(OutgoingRequestsLogAdmin):
+    def change_view(
+        self,
+        request: HttpRequest,
+        object_id: str,
+        form_url: str = "",
+        extra_context: dict[str, Any] | None = None,
+    ) -> HttpResponse:
+        outgoing_request_log = self.get_object(request, object_id)
+        if outgoing_request_log is None:
+            raise Http404(f"No {self.model._meta.object_name} matches the given query.")
```

Comment:
missing coverage

Reply:
that doesn't absolve from testing bad ID lookups :) a decorator is an implementation detail